index.html

```diff
@@ -77,18 +78,25 @@ <h1> <b>|| Selena Marie Gomez ||</b></h1>
 
      <div class="row row-offcanvas row-offcanvas-right">
 
-        <div class="col-xs-12 col-sm-9 " class="float-none1">
+        <div class="col-xs-12 col-sm-9 " id="float-none1">
```

Comment:
This needs to be a class. Please sync your fork and then make your changes on top of it. You would need to `rebase`